docker/gemini.dockerfile

```diff
@@ -10,19 +10,24 @@ USER root
 RUN conda create --name py2 python=2.7 && \
     bash -c "source activate py2 && conda install --yes -c conda-forge -c bioconda gemini=0.30.2-0 vt=2015.11.10 variant-effect-predictor=87-0"
 
+
+RUN curl -s -o /usr/local/bin/pull-tutorial.sh https://raw.githubusercontent.com/statgenetics/statgen-courses/master/src/pull-tutorial.sh
+RUN chmod a+x /usr/local/bin/pull-tutorial.sh
+
+RUN sed -i '2 i \
+	pull-tutorial.sh gemini & \
+	'  /usr/local/bin/start-notebook.sh
+
+
 #Download datafiles 
 #Update the exercise handout
 
 USER jovyan
 
-RUN curl -fsSL http://statgen.us/files/gemini-tutorial-data.tar.bz2 -o gemini.tar.bz2 && tar jxvf gemini.tar.bz2 && rm -f gemini.tar.bz2
+RUN ( cd $HOME/work && curl -fsSL http://statgen.us/files/gemini-tutorial-data.tar.bz2 -o gemini.tar.bz2 && tar jxvf gemini.tar.bz2 && rm -f gemini.tar.bz2 )
 
 RUN echo "source activate py2" >> $HOME/.bashrc
 
-RUN mkdir -p $HOME/.gemini $HOME/annotation_db && echo "annotation_dir: $HOME/annotation_db" > $HOME/.gemini/gemini-config.yaml
-
-RUN printf "This folder is meant for gemini annotation resource databases,\nas is configured in ~/.gemini/gemini-config.yaml\nThese annotations were used along with VCF files as input to generate the database files in the tutorial.\n" > $HOME/annotation_db/README.md
-
-ARG DUMMY=unknown
+RUN mkdir -p $HOME/.gemini $HOME/work/annotation_db && echo "annotation_dir: $HOME/work/annotation_db" > $HOME/.gemini/gemini-config.yaml
```

Comment:
We should make annotation_db sit on `$HOME` and use this configuration for it. This is similar to annovar exercise but there we used symbolic link